example-data.json

```diff
@@ -0,0 +1,14 @@
+{
+  "data": {
+    "type": "reviews",
+    "id": 1
```

Comment:
According to the spec data.id must be a string. https://jsonapi.org/format/#document-resource-object-identification